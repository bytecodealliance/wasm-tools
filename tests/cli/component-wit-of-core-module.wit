// RUN: component embed --dummy % | component wit

package a:b;

world foo {
  import x: func();
  export y: func();
}
