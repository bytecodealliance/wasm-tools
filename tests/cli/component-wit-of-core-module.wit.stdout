package root:root;

world root {
  import x: func();

  export y: func();
}
package a:b {
  world foo {
    import x: func();

    export y: func();
  }
}
